doc/docs/master/enterprise/auth/oidc/configure-auth0.md

```diff
@@ -0,0 +1,188 @@
+
+* Pachyderm Enterprise 1.11.x or later. The enterprise token must be
+  activated by running `pachctl enterprise activate`.
+  Check the status of your license by running:
+
+  ```bash
+  pachctl enterprise get-state
+  ```
+
+  For more information, see [Activate Pachyderm Enterprise Edition](../../../deployment/#activate-pachyderm-enterprise-edition).
+
+* An account at https://auth0.com . It could be a new or an existing
+account, either paid or free.
+
+## Configure Auth0
+
+If you do not have an Auth0 account, you need to sign up for one
+at https://auth0.com . Then, you need to add Pachyderm as an application
+and configure it to work with Auth0.
+
+To configure Auth0, complete the following steps:
+
+1. Log in to your Auth0 account.
+1. In the **Applications**, click **Create Application**.
+1. Type the name of your application, such as **Pachyderm**.
+1. In the application type, select **Regular Web Application**.
+1. Click **Create**.
+1. Go to the application settings.
+1. Scroll down and click **Show Advanced Settings**.
+1. Select **Grant Types**.
+1. Verify that **Implicit**, **Authorization Code**, **Refresh Token**, and
+**Client Credentials** are selected.
+
+   ![Auth0 Grant Settings](../../../assets/images/s_auth0_grant_settings.png)
+
+1. In the **Allowed Callback URLs**, add the Pachyderm callback link in the
+following format:
+
+   ```bash
+   http://<ip>:30657/authorization-code/callback
+   ```
+
+   The IP address is the address of your Pachyderm host. For example,
+   if you are running Pachyderm in Minikube, you can find the IP
+   address by running `minikube ip`.
+
+1. Proceed to [Configure Pachyderm Auth](#configure-pachyderm-auth).
+
+## Configure Pachyderm Auth
+
+After you have configured a Pachyderm application in Auth0, you
+need to create a Pachyderm OIDC config with the Auth0 parameters.
+All the required parameters, such as `client_id`, `client_secret`, 
+and othersi, are located on the application settings screen. In addition, OIDC
+settings are exposed at https://appication-domain/.well-known/openid-configuration.
+
+To configure Pachyderm Auth, complete the following steps:
+
+1. Check the status of your license by running:
+
+   ```bash
+   pachctl enterprise get-state
+   ```
+
+   You must have an active enterprise token to proceed.
+
+1. Go to the terminal and forward the `pachd` pod to the OIDC port:
+
+   1. Get the `pachd` pod ID:
+
+      ```bash
+      kubectl get pod
+      ```
+
+      **Example system response:**
+
+      ```bash
+      dash-5768cb7d98-j6cgt       2/2     Running   0          4h2m
+      etcd-56d897697-xzsqr        1/1     Running   0          4h2m
+      keycloak-857c59449b-htg99   1/1     Running   0          4h6m
+      pachd-79f7f68c65-9qs8g      1/1     Running   0          4h2m
+      ```
+
+   1. Forward the `pachd` pod to the OIDC port:
+
+      **Example:**
+
+      ```bash
+      kubectl port-forward pachd-79f7f68c65-9qs8g 30657
```

Comment:
did you need to do this for it to work? 

i added automatic support for forwarding to 30657 in pachd that works on my machine, so hopefully is not necessary

Reply:
I had to do it. Maybe let's keep it, for now, to be on the safe side?

Reply:
that makes sense